style.css

```diff
@@ -0,0 +1,458 @@
+/* stylelint-disable csstree/validator */
+
+/* stylelint-disable-next-line block-opening-brace-space-before */
+h1 {
+  width: 327px;
+  height: 150px;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
```

Comment:
- [ ] According to Microverse, You are not meant to have any inline disable rule in your project, The linters are here to make sure you follow the best practices and conventions, I can see you have a lot of inline linter disable comments, this makes your code base look a bit unprofessional and also against the p=Microverse requirement, I'll suggest you go through this file and remove all the linters disabel rules and try to fix the issues. :smile: :wave: :100: